crop_classification/inference.py

```diff
@@ -0,0 +1,111 @@
+from cropharvest.engineer import Engineer
+from datetime import datetime
+from pathlib import Path
+import numpy as np
+import xarray as xr
+import pandas as pd
+import re
+import torch
+
+from typing import Dict, List, Optional, Tuple, Union
+
+
+class Inference:
+    """
+    Class for running inference using either a pytorch checkpoint model or
+    a pytorch jit model on a single tif file.
+    """
+
+    def __init__(self, model, device: Optional[torch.device] = None):
+        self.model = model
+        self.device = device
+
+        # TODO: normalizing_dict is property of the model in crop-mask should this be the same here?
```

Comment:
In `crop-mask` when initializing the `Model` during training I compute (or pull from cache) the normalizing_dict and have an attribute: `self.normalizing_dict_jit: Dict[str, List[float]]`. Since the dictionary only has primitives it can be saved inside the jit model and use when the model is loaded. Would something like that make sense in this repo? How is the normalizing dict computed / used here?

Reply:
We load the normalizing dict from the `cropharvest` normalizing dict (so its loaded as part of the cropharvest datasets).

I think it would be nice to have a check somehow; by default we expect a normalizing dict, and the model should throw a warning if it doesn't have one